cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIConfig.java

```diff
@@ -66,37 +70,44 @@
     private final boolean tlsEnabled;
 
     /**
-     * User name if authentication is configured in the Controller.
+     * User name if authentication is configured in the CLI.
      */
     @Getter
     private final String userName;
 
     /**
-     * Password if authentication is configured in the Controller.
+     * Password if authentication is configured in the CLI.
      */
     @Getter
     private final String password;
 
     /**
-     * Truststore if TLS is configured in the Controller.
+     * Truststore if TLS is configured in the CLI.
      */
     @Getter
     private final String truststore;
 
+    /**
+     * Truststore access token ttl in seconds if TLS is configured in the CLI.
+     */
+    @Getter
+    private final Duration accessTokenTtlInSeconds;
```

Comment:
rename this to "accessTokenTtl". `Duration` holds time, and you can use it to express it in any time unit you want.

Also fix the Javadoc please.